<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsLayoutDesignerBase</class>
 <widget class="QMainWindow" name="QgsLayoutDesignerBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>609</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="mViewFrame">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="mStatusBar"/>
  <widget class="QToolBar" name="mLayoutToolbar">
   <property name="windowTitle">
    <string>Layout</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="mToolsToolbar">
   <property name="windowTitle">
    <string>Toolbox</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="mActionPan"/>
   <addaction name="mActionZoomTool"/>
   <addaction name="mActionSelectMoveItem"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="mLayoutMenu">
    <property name="title">
     <string>&amp;Layout</string>
    </property>
    <addaction name="mActionLayoutProperties"/>
    <addaction name="mActionAddPages"/>
    <addaction name="separator"/>
    <addaction name="mActionClose"/>
   </widget>
   <widget class="QMenu" name="mItemMenu">
    <property name="title">
     <string>&amp;Items</string>
    </property>
   </widget>
   <widget class="QMenu" name="mMenuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="mToolbarMenu">
     <property name="title">
      <string>&amp;Toolbars</string>
     </property>
    </widget>
    <widget class="QMenu" name="mPanelsMenu">
     <property name="title">
      <string>&amp;Panels</string>
     </property>
    </widget>
    <addaction name="mActionZoomIn"/>
    <addaction name="mActionZoomOut"/>
    <addaction name="mActionZoomActual"/>
    <addaction name="mActionZoomAll"/>
    <addaction name="mActionZoomToWidth"/>
    <addaction name="separator"/>
    <addaction name="mActionShowGrid"/>
    <addaction name="mActionSnapGrid"/>
    <addaction name="separator"/>
    <addaction name="mActionShowGuides"/>
    <addaction name="mActionSnapGuides"/>
    <addaction name="mActionManageGuides"/>
    <addaction name="mActionClearGuides"/>
    <addaction name="separator"/>
    <addaction name="mActionShowRulers"/>
    <addaction name="separator"/>
    <addaction name="mToolbarMenu"/>
    <addaction name="mPanelsMenu"/>
    <addaction name="mActionToggleFullScreen"/>
   </widget>
   <addaction name="mLayoutMenu"/>
   <addaction name="mMenuView"/>
   <addaction name="mItemMenu"/>
  </widget>
  <widget class="QToolBar" name="mNavigationToolbar">
   <property name="windowTitle">
    <string>Navigation</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mActionZoomIn"/>
   <addaction name="mActionZoomOut"/>
   <addaction name="mActionZoomActual"/>
   <addaction name="mActionZoomAll"/>
  </widget>
  <action name="mActionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close designer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="mActionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionPan.svg</normaloff>:/images/themes/default/mActionPan.svg</iconset>
   </property>
   <property name="text">
    <string>Pan Layout</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="mActionZoomTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionZoomToArea.svg</normaloff>:/images/themes/default/mActionZoomToArea.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="toolTip">
    <string>Zoom</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
  <action name="mActionSelectMoveItem">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionSelect.svg</normaloff>:/images/themes/default/mActionSelect.svg</iconset>
   </property>
   <property name="text">
    <string>Move &amp;Item</string>
   </property>
   <property name="toolTip">
    <string>Select/Move item</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="mActionZoomAll">
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionZoomFullExtent.svg</normaloff>:/images/themes/default/mActionZoomFullExtent.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Full</string>
   </property>
   <property name="toolTip">
    <string>Zoom full</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="mActionZoomIn">
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionZoomIn.svg</normaloff>:/images/themes/default/mActionZoomIn.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="mActionZoomOut">
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionZoomOut.svg</normaloff>:/images/themes/default/mActionZoomOut.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="mActionZoomActual">
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionZoomActual.svg</normaloff>:/images/themes/default/mActionZoomActual.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom to &amp;100%</string>
   </property>
   <property name="toolTip">
    <string>Zoom to 100%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="mActionZoomToWidth">
   <property name="text">
    <string>Zoom to Width</string>
   </property>
  </action>
  <action name="mActionShowRulers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Ru&amp;lers</string>
   </property>
   <property name="toolTip">
    <string>Show rulers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="mActionToggleFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle Full Scr&amp;een</string>
   </property>
   <property name="toolTip">
    <string>Toggle full screen mode</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="mActionAddPages">
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionFileNew.svg</normaloff>:/images/themes/default/mActionFileNew.svg</iconset>
   </property>
   <property name="text">
    <string>Add Pagesâ€¦</string>
   </property>
  </action>
  <action name="mActionShowGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/vector_grid.png</normaloff>:/images/themes/default/vector_grid.png</iconset>
   </property>
   <property name="text">
    <string>Show &amp;Grid</string>
   </property>
   <property name="toolTip">
    <string>Show grid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+'</string>
   </property>
  </action>
  <action name="mActionSnapGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S&amp;nap to Grid</string>
   </property>
   <property name="toolTip">
    <string>Snap to grid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+'</string>
   </property>
  </action>
  <action name="mActionManageGuides">
   <property name="text">
    <string>Manage Guidesâ€¦</string>
   </property>
  </action>
  <action name="mActionShowGuides">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show G&amp;uides</string>
   </property>
   <property name="toolTip">
    <string>Show guides</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+;</string>
   </property>
  </action>
  <action name="mActionSnapGuides">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Snap to Guides</string>
   </property>
   <property name="toolTip">
    <string>Snap to guides</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+;</string>
   </property>
  </action>
  <action name="mActionClearGuides">
   <property name="text">
    <string>&amp;Clear Guides</string>
   </property>
   <property name="toolTip">
    <string>Clear guides</string>
   </property>
  </action>
  <action name="mActionLayoutProperties">
   <property name="text">
    <string>Layout Propertiesâ€¦</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
